packages/metro/src/ModuleGraph/worker/collectDependencies.js

```diff
@@ -251,6 +288,127 @@ function collectDependencies<TSplitCondition = void>(
   };
 }
 
+/** Extract args passed to the `require.context` method. */
+function getRequireContextArgs(
+  path: NodePath<CallExpression>,
+): [string, RequireContextParams] {
+  const args = path.get('arguments');
+
+  let directory: string;
+  if (!Array.isArray(args) || args.length < 1) {
+    throw new InvalidRequireCallError(path);
+  } else {
+    const argNode = args[0].node;
+    if (argNode.type !== 'StringLiteral') {
+      throw new InvalidRequireCallError(
+        argNode,
+        `First argument of \`require.context\` should be a string denoting the directory to require, instead found node of type: ${argNode.type}.`,
```

Comment:
`InvalidRequireCallError`'s [constructor](https://github.com/facebook/metro/blob/57108f84338017b0c7a2d4bc38a3a3cc814df807/packages/metro/src/ModuleGraph/worker/collectDependencies.js#L582-L590) doesn't take a second argument, I assume you're planning to change it? (Comment mostly for my benefit so I don't forget in the final review pass)